internal/connector/internal/services/connector_types.go

```diff
@@ -232,7 +235,7 @@ func (cts *connectorTypesService) List(listArgs *services.ListArguments) (dbapi.
 	// set total, limit and paging (based on https://gitlab.cee.redhat.com/service/api-guidelines#user-content-paging)
 	total := int64(pagingMeta.Total)
 	dbConn.Model(&resourceList).Count(&total)
-	pagingMeta.Total = int(total)
+	pagingMeta.Total = int(dbConn.RowsAffected)
```

Comment:
This is not needed as the SQL count in total already contains the total number of records from the query.

Reply:
This is needed the total returns the wrong value when rows affected are only one